subjects/mobile-dev/map-markers/audit/README.md

```diff
@@ -1,35 +1,37 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
-###### Does the application run without crashes?
+###### Does the app run without crashing?
 
-###### Does the application have Bottom tabbar?
+###### Does the app have `Bottom TabBar`?
```

Comment:
In the subject is requested to have a `TapBar`

Reply:
done